generics/helpers/common_handler.js

```diff
@@ -705,7 +705,7 @@ const createChart = async function (chartData, imgPath) {
  * @param {string} userId - The ID of the user initiating the upload.
  * @param {string} folderPath - The path to the folder where the PDF file is located.
  */
-const uploadPdfToCloud = async function (fileName, userId, folderPath) {
+exports.uploadPdfToCloud = async function (fileName, userId, folderPath) {
```

Comment:
@Prajwal17Tunerlabs why change this? Use it how other functions from here are called. use file used for cloud related functions?

Reply:
this api is used for reports generation and save the pdf to cloud, for reports there is another function in this file which later calls uploadPdfToCloud to upload pdfs , until this time this function was not exported, now since i had the requirement of ulploading pdf and svg files i reused this function